application/cmd/requester/main.go

```diff
@@ -0,0 +1,18 @@
+//  Copyright 2020 Insolar Network Ltd.
+//  All rights reserved.
+//  This material is licensed under the Insolar License version 1.0,
+//  available at https://github.com/insolar/insolar/blob/master/master/LICENSE.md.
```

Comment:
Update to `// available at https://github.com/insolar/insolar/blob/master/LICENSE.md.`